pkgs/development/tools/flamegraph/default.nix

```diff
@@ -0,0 +1,27 @@
+{ stdenv, fetchFromGitHub, perl }:
+
+stdenv.mkDerivation {
+  name = "FlameGraph-2015-10-10";
+
+  src = fetchFromGitHub {
+    owner = "brendangregg";
+    repo = "FlameGraph";
+    rev = "182b24fb635345d48c91ed1de58a08b620312f3d";
+    sha256 = "1djz0wl8202a6j87ka9j3d8iw3bli056lrn73gv2i65p16rwk9kc";
+  };
+
+  buildInputs = [ perl ];
+
+  installPhase = ''
+    mkdir -p $out/bin
+    for x in $src/*.pl $src/*.awk $src/dev/*.pl $src/dev/*.d; do
```

Comment:
I don't know. I don't have access to dtrace to test, since dtrace does not support linux. It's possible that they run on macs.

if it's a sticking point, I'm just as happy to not install the *.d files at all and leave a comment about it in the .nix file.